libraries/fbx/src/FBXSerializer.cpp

```diff
@@ -1299,35 +1321,64 @@ HFMModel* FBXSerializer::extractHFMModel(const hifi::VariantHash& mapping, const
         joint.rotationMin = fbxModel.rotationMin;
         joint.rotationMax = fbxModel.rotationMax;
 
-        joint.hasGeometricOffset = fbxModel.hasGeometricOffset;
-        joint.geometricTranslation = fbxModel.geometricTranslation;
-        joint.geometricRotation = fbxModel.geometricRotation;
-        joint.geometricScaling = fbxModel.geometricScaling;
+        if (fbxModel.hasGeometricOffset) {
+            joint.geometricOffset = createMatFromScaleQuatAndPos(fbxModel.geometricScaling, fbxModel.geometricRotation, fbxModel.geometricTranslation);
+        }
         joint.isSkeletonJoint = fbxModel.isLimbNode;
         hfmModel.hasSkeletonJoints = (hfmModel.hasSkeletonJoints || joint.isSkeletonJoint);
+
+        joint.name = fbxModel.name;
+
+        joint.bindTransformFoundInCluster = false;
+
+        // With the basic joint information, we can start to calculate compound transform information
+        // modelIDs is ordered from parent to children, so we can safely get parent transforms from earlier joints as we iterate
+
+        // Make adjustments to the static joint properties, and pre-calculate static transforms
+
         if (applyUpAxisZRotation && joint.parentIndex == -1) {
             joint.rotation *= upAxisZRotation;
             joint.translation = upAxisZRotation * joint.translation;
         }
+
         glm::quat combinedRotation = joint.preRotation * joint.rotation * joint.postRotation;
+        glm::mat4 localTransform = glm::translate(joint.translation) * joint.preTransform * glm::mat4_cast(combinedRotation) * joint.postTransform;
+
         if (joint.parentIndex == -1) {
-            joint.transform = hfmModel.offset * glm::translate(joint.translation) * joint.preTransform *
-                glm::mat4_cast(combinedRotation) * joint.postTransform;
+            joint.transform = localTransform * joint.geometricOffset;
+            joint.globalTransform = hfmModel.offset * localTransform * joint.geometricOffset;
+            joint.globalTransformForChildren = hfmModel.offset * localTransform;
```

Comment:
this is basically the fix...when hifi did the instancing PR, they removed support for geometricOffset

I don't know if there's an analogous property in GLTF so for now this (globalTransformForChildren) is only used here so that each joint has the correct globalTransform (including geometricOffset), but it doesn't pass geometricOffset down to its children

Reply:
The top level object in glTF is the "scene". The scene has one or more "nodes". Each node may have a transform plus one or more meshes, and it can have other nodes as children (e.g., to form a skeleton).